### ISSUE TITLE
Advanced Data Solution ( so far all others also) not opening in VS2015 or VS2013

### ISSUE THREAD

**Issue by TeeRodFl** [xamarin/mobile-samples#39]
**VS2015 Reports the following: **"_D:\Solutions 2015\mobile-samples-master\mobile-samples-master\ContentControls\iOSContentControls\iOSContentControls.csproj : error  : The project file 'D:\Solutions 2015\mobile-samples-master\mobile-samples-master\ContentControls\iOSContentControls\iOSContentControls.csproj' cannot be opened."
The project type is not supported by this installation._

I found this error related identification on a another Xamarin GitHub project.  Go to the bottom to see the resolution for this Visual Studio error. [https://groups.google.com/forum/#!msg/SpecFlow/D9B66kUlfJc/tqiCR1WtWncJ] "**one or more projects in the solution were not loaded correctly"**

Apparently nobody is using Xamarin Visual Studio 2015 2013 or 2010 but me.

**I have attached a screen shot of the error and the specific projects**, I would imagine that all of the master projects solutions have the same problem, so if you are wondering why no Microsoft Visual Studio users are commenting it is probably because they cant open the projects.  It may only be the project files because the sln seems to find the project with no trouble.

ScreenShot
Looking forward to using the samples!

Best regards 
Robert Morgan

**Comment by TeeRodFl** [comment]
ScreenShot
![mobile-samples-master_dataaccess_advanced_error](https://cloud.githubusercontent.com/assets/13560143/14939658/b30c189a-0f1e-11e6-8c96-83caa8c7e833.JPG)

**Comment by TeeRodFl** [comment]
Closed by mistake, this issue is active and new
Robert Morgan

**Comment by conceptdev** [comment]
Xamarin is no longer supported in Visual Studio 2010, however 2012, 2013, and 2015 are supported. 
Please confirm that you have correctly installed Xamarin - you must download and install the Xamarin plugin with versions prior to 2015, and in 2015 you must tick a box during install to add Xamarin functionality. To confirm, check the Tools menu to see Xamarin-related menu items (and also Xamarin sections in Tools > Options).

**Comment by TeeRodFl** [comment]
Thank you for the response, I have been working with Xamarin for several months now and believe I fully understand the installation requirements I don't use 2012 any longer 2010 was a type-o.  I can only assume you haven't downloaded the current version and opened it in VS2015.  I have included screen shots of my install for your reference.  I have downloaded 20+  Xamarin projects from GitHub with no issue though.

Regards
Robert
![xamarin proof of life 1](https://cloud.githubusercontent.com/assets/13560143/14939889/49e4673a-0f2a-11e6-81fb-ffec4dd2dd3d.JPG)
![xamarin proof of life 2](https://cloud.githubusercontent.com/assets/13560143/14939891/508724d8-0f2a-11e6-8065-961d0730382a.JPG)
![xamarin proof of life 3](https://cloud.githubusercontent.com/assets/13560143/14939892/54f52254-0f2a-11e6-9fdb-e063a24be4ed.jpg)

**Comment by conceptdev** [comment]
I see - my misunderstanding. I understood "(so far all others also) not opening" to mean that you could not successfully load _any_ Xamarin project, hence my suggestion to check the installation.

Is there any additional information you can provide about your configuration? I am able to open the **DataAccess_Adv** solution successfully in Visual Studio 2015:

![dataaccess_adv](https://cloud.githubusercontent.com/assets/199373/14939959/952d6596-0f13-11e6-8e54-9be7fab306f2.png)

In particular can you confirm which versions of Visual Studio are experiencing the error you raised in the initial issue - the screenshot seems to show an older VS icon and appears to be running on Windows 7?

**Comment by TeeRodFl** [comment]
Once you confirmed it opened for you, I re-downloaded and overwrote the master project. As you said it opened as expected. I appreciated your quick response to the issue.  I can only figure the first decompress had a corruption issue that WinRar did not report back to me.  

I am looking forward to learning your techniques for handling database issues and enhancements,  I am an experienced SQL ADO/Linq programmer but SQLite and Android opens new avenues for me to learn that I am anxious to explore.  I saw the document online and realized you have far more than just database in the package, I have randomly opened several other solutions and they opened as expected.

Thanks again for the response and eventual resolution on my side, go figure, a comedy of errors.

BTW, my system is Windows 7 development with testing on Windows 10 and VS2015 android API 23 - 16 no IOs testing right now

Regards 
Robert
